### ISSUE TITLE
"12p" and "12pm" parsed to BYHOUR=12 and BYHOUR=24, respectively

### ISSUE THREAD

**Issue by itdaniher** [kvh/recurrent#14]
```python
>>> recurrent.RecurringEvent().parse("daily at 12p")
'RRULE:BYHOUR=12;BYMINUTE=0;INTERVAL=1;FREQ=DAILY'
>>> recurrent.RecurringEvent().parse("daily at 12pm")
'RRULE:BYHOUR=24;BYMINUTE=0;INTERVAL=1;FREQ=DAILY'
````

expected result is both parse to `BYHOUR=12`

**Comment by snoopyjc** [comment]
Fixed in https://github.com/kvh/recurrent/pull/19

**Comment by snoopyjc** [comment]
Fixed in 0.3.0.
